.ebextensions/03_autoscaling.config

```diff
@@ -0,0 +1,6 @@
+Resources:
+  AWSEBAutoScalingGroup:
+    Type: AWS::AutoScaling::AutoScalingGroup
+    Properties:
+      HealthCheckType: ELB
+      HealthCheckGracePeriod: 900
```

Comment:
do we really need/want 15 minute lead time for the health check grace period? that seems like a lot and could lead to a buildup of dead ec2s still?